5_People/salary-system.md

```diff
@@ -39,7 +39,7 @@ Once a year, in January, during our [bi-annual IRL](https://blog.niteo.co/irls/)
 
 ### Location Factor
 
-Location factor takes into account the living costs. We determine them by comparing Numbeo's [cost of living between San Francisco and the Nitean's city](https://www.numbeo.com/cost-of-living/compare_cities.jsp?country1=United+States&city1=San+Francisco%2C+CA&country2=Germany&city2=Berlin) and calculating the *Numbeo Location Ratio*.
+Location factor takes into account the living costs. We determine them by comparing Numbeo's [cost of living between San Francisco and the Nitean's city](https://www.numbeo.com/cost-of-living/compare_cities.jsp?country1=United+States&city1=San+Francisco%2C+CA&country2=Germany&city2=Berlin) and calculating the *Numbeo Location Ratio*. For the purposes of the Salary System, we take your official permanent residence (address stated in your passport), and do not change it for seasonal relocations or longer travel.
```

Comment:
What about temporary residence? If I am not mistaken I can have a temporary residence in my passport? Or is this just a construct of epic Slovenian bureaucracy?

Reply:
http://www.mnz.gov.si/mnz_za_vas/osebni_dokumenti_in_prebivalisce/prijava_in_odjava_prebivalisca/zacasno_prebivalisce/

Reply:
Seems this does not affect "address in passport". So, meh.